edx_proctoring/tests/test_views.py

```diff
@@ -556,6 +556,52 @@ def test_start_exam_callback(self):
         self.assertEqual(attempt['status'], "started")
         self.assertFalse(attempt['status'] == "ready_to_start")
 
+    def test_start_exam_callback_when_download_software_clicked(self):
+        """
+        Test that hitting software secure callback URL twice when the attempt state begins at
+        'download_software_clicked' does not change the state from 'started' back to 'ready to start'
+        """
+        # Create an exam.
+        proctored_exam = ProctoredExam.objects.create(
```

Comment:
How about pulling the exam creation logic out into a private helper that can be shared by this test and the one above?